fe/fe-core/src/main/java/org/apache/doris/analysis/LabelName.java

```diff
@@ -32,6 +32,7 @@
 import java.io.DataOutput;
 import java.io.IOException;
 
+// TODO(tsy): maybe better to rename as `LoadLabel`
```

Comment:
Aggress with it.

Reply:
Maybe we should make another PR, that rename may affaect lots of unrelated files.